_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -28,15 +28,10 @@ This rule applies to any [WAI-ARIA 1.1 state or property](https://www.w3.org/TR/
 
 Each test target has a valid value according to its [WAI-ARIA 1.1 value type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value).
 
-For value types `ID Reference` and `ID Reference List` for [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.1/#requiredState) at least one of the elements with the given ids exists in the same [document tree](https://dom.spec.whatwg.org/#document-trees) or [shadow tree](https://dom.spec.whatwg.org/#shadow-trees) as the element that specifies the target attribute.
+For value types `ID Reference` and `ID Reference List` for [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.1/#requiredState) at least one of the elements with the given ids exists in the same [document tree](https://dom.spec.whatwg.org/#document-trees) or in the same [shadow tree](https://dom.spec.whatwg.org/#shadow-trees) as the element that specifies the target attribute.
```

Comment:
I don't think this is accurate. This says the reference must exist, and then in the background it says, yes but only if the prop is required. We should update this text to only require the element to exist if the attribute is required.

Reply:
It says "for WAI-ARIA required properties". Isn't that enough?